vendor/github.com/flosch/pongo2/lexer.go

```diff
@@ -70,7 +70,7 @@ type lexer struct {
 func (t *Token) String() string {
 	val := t.Val
 	if len(val) > 1000 {
-		val = fmt.Sprintf("%s...%s", val[:10], val[len(val)-5:len(val)])
```

Comment:
Gofmt for gohan source is ok, but why do we enforce gofmt for vendor libraries which we do not own?